Jenkinsfile

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'manuals-frontend'
+
+node {
+  def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
+
+  try {
+    stage('Checkout') {
+      checkout scm
+      govuk.cleanupGit()
+      govuk.mergeMasterBranch()
+      govuk.contentSchemaDependency()
+    }
+
+    stage('Bundle') {
+      govuk.bundleApp()
+    }
+
+    stage('Linter') {
+      govuk.rubyLinter()
+    }
+
+    stage('Tests') {
+      govuk.setupDb()
+      govuk.setEnvar('RAILS_ENV', 'test')
+      govuk.runTests()
+    }
+
+    stage("Precompile assets") {
+      govuk.precompileAssets()
+    }
+
+    stage('Push release tag') {
```

Comment:
Making these entire stages conditional on building master (as in [1]) is good, as it makes the Jenkins interface less confusing when building other branches, as it doesn't show that it has pushed a release tag, when it hasn't.

1: https://github.com/alphagov/gds-api-adapters/blob/master/Jenkinsfile#L121